esmvaltool/diag_scripts/weighting/climwip/core_functions.py

```diff
@@ -108,9 +120,32 @@ def compute_overall_mean(dataset: 'xr.Dataset',
 
 
 def combine_ensemble_members(
-        dataset: Union['xr.DataArray', None]) -> (
-            Union['xr.DataArray', None], dict):
-    """Combine ensemble members of the same model."""
+        dataset: Union['xr.DataArray', None],
+        dimns: Union[str, list] = 'model_ensemble',
+) -> (Union['xr.DataArray', None], dict):
+    """Combine ensemble members of the same model.
+
+    Parameters
+    ----------
+    dataset : None or data_array, shape (N,) or (N, N)
+        A vector containing model-observations distances or a matirx containing
+        model-model distances.
+    dimns : string or list of up to two strings
```

Comment:
It is clearer to write dimensions instead of dimns, imo.

(or perhaps dim / dims if you prefer to stick with xarray nomenclature)

```suggestion
    dimensions: string or list of up to two strings
```